setup.py

```diff
@@ -0,0 +1,38 @@
+import setuptools
+from setuptools import setup
+from distutils.util import convert_path
+import os
+
+main_ns = {}
+ver_path = convert_path('src/version.py')
+with open(ver_path) as ver_file:
+    exec(ver_file.read(), main_ns)
+VER = main_ns['__version__']
+
+requirementPath = os.path.dirname(os.path.realpath(__file__)) + '/requirements.txt'
```

Comment:
is there a reason for not working with relative paths?

Reply:
Saw it's common in other repos. Safer this way I guess.